### DISCUSSION TITLE
Error: stash_req is missing or incompatible in items namelist.

### DISCUSSION THREAD

**Discussion by mo-abodas** [MetOffice/simulation-systems#107] [UM]
Hello,

I'm getting the following error in one of my runs:

????????????????????????????????????????????????????????????????????????????????
???!!!???!!!???!!!???!!!???!!!       ERROR        ???!!!???!!!???!!!???!!!???!!!
?  Error code: 10
?  Error from routine: READ_NML_ITEMS
?  Error message: stash_req is missing or incompatible in items namelist.
?        Please check entries.
?  Error from processor: 0
?  Error number: 12
????????????????????????????????????????????????????????????????????????????????

Where should I look to get more information about the offending stash?
Thanks!

**Comment by mcdalvi** [comment]
Hi,

In case you have not solved this yet - check the _[namelist:items]_ entries in the app/um/rose-app.conf (and any app/um/opt/rose-app-XXXX.conf files that are being used).
This namelist category is used to specify ancillary files and the fields to read from each file (as well as initialisation of some model fields, if not from the start dump). 
Each namelist:items(NNNNN)] entry should have a 'stash_req' keyword that specifies the stashcode corresponding to input ancillary field. There might be a syntax error (spaces, or multiple entries not separated by comma) in any new entries.

Hope that helps

**Reply by mo-abodas** [reply]
Thanks @mcdalvi , it seems that one of the options files I'm using has an incomplete stash item. I haven't fixed it yet, but it looks like that should be the cause of the issue.
